src/LineBlockTracer.h

```diff
@@ -45,12 +45,10 @@ class cLineBlockTracer:
 	cLineBlockTracer(cWorld & a_World, cCallbacks & a_Callbacks);
 
 	/** Traces one line between Start and End; returns true if the entire line was traced (until OnNoMoreHits()) */
-	bool Trace(double a_StartX, double a_StartY, double a_StartZ, double a_EndX, double a_EndY, double a_EndZ);
+	bool Trace(const Vector3d & a_Start, const Vector3d & a_End);
```

Comment:
`Vector3` should be passed by value:
```suggestion
	bool Trace(Vector3d a_Start, Vector3d a_End);
```